### PR TITLE
RandomRepositioning: DemandFollowingRepositioningManager

### PR DESCRIPTION
**Pull request by REASY**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1992)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -44,7 +42,7 @@ systemProp.org.gradle.internal.http.socketTimeout=180000
 
 #r5.24xlarge (96/768) -> 3 -> $6.048 per Hour
 #r5d.24xlarge (96/768) -> 3 -> $6.912 per Hour
-instanceType=c5.18xlarge
+instanceType=m4.16xlarge
```
Comment (wrashid) [inline gradle.properties]: c5.18xlarge

File: gradle.properties [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
 #executeArgs=['--config', 'production/application-sfbay/experimentsFall2018/fall18-calib-run1.conf']
 beamBatch=false
 shutdownWait=15
 # shutdownBehaviour = stop | terminate
 shutdownBehaviour=stop
 s3Backup=true
-maxRAM=140g
-#storageSize (in GiB) =  any number between 64 and 512. default 64
-storageSize=64
+maxRAM=200g
```
Comment (wrashid) [inline gradle.properties]: 140g

File: src/main/resources/beam-template.conf [hunk 3]
```diff
@@ -120,8 +120,11 @@ beam.agentsim.agents.rideHail.refuelLocationType="AtTAZCenter"
 beam.agentsim.agents.rideHail.rideHailManager.radiusInMeters="double | 5000"
 beam.agentsim.agents.rideHail.allocationManager.name="DEFAULT_MANAGER"
 beam.agentsim.agents.rideHail.allocationManager.requestBufferTimeoutInSeconds="int | 0"
-beam.agentsim.agents.rideHail.allocationManager.repositionTimeoutInSeconds="int | 0"
-beam.agentsim.agents.rideHail.allocationManager.randomRepositioning.repositioningShare="double | 0.1"
+beam.agentsim.agents.rideHail.repositioningManager.name="NoOpRepositioningManager"
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: NO_REPOSITIONING

File: src/main/scala/beam/sim/BeamSim.scala [hunk 4]
```diff
@@ -84,6 +86,8 @@ class BeamSim @Inject()(
   var metricsPrinter: ActorRef = actorSystem.actorOf(MetricsPrinter.props())
   val summaryData = new mutable.HashMap[String, mutable.Map[Int, Double]]()
 
+  val rhuc: RideHailUtilizationCollector = new RideHailUtilizationCollector(beamServices)
```
Comment (wrashid) [inline src/main/scala/beam/sim/BeamSim.scala]: rideHailUtilizationCollector

**Comment by REASY** [comment]
test!

**Comment by REASY** [comment]
test!

**Pull request merged**